documentation/example_dataset/manifest.json

```diff
@@ -1,9 +1,27 @@
 {
   "frames": ["frame_0.png", "frame_1.png", "frame_2.png", "frame_3.png", "frame_4.png"],
-  "features": { "feature_0": "feature_0.json", "feature_1": "feature_1.json" },
+  "features": {
+    "Continuous Feature": "feature_0.json",
+    "Discrete Feature": "feature_1.json",
+    "Categorical Feature": "feature_2.json"
+  },
   "bounds": "bounds.json",
   "times": "times.json",
   "tracks": "tracks.json",
   "centroids": "centroids.json",
-  "outliers": "outliers.json"
+  "outliers": "outliers.json",
+  "featureMetadata": {
+    "Continuous Feature": {
+      "units": "°F",
+      "type": "continuous"
+    },
+    "Discrete Feature": {
+      "units": "nodes",
+      "type": "discrete"
+    },
+    "Categorical Feature": {
+      "type": "categorical",
+      "categories": ["Spicy", "Sour", "Bitter", "Sweet", "Dry"]
+    }
```

Comment:
seeing this example, it looks weird to separate the metadata from the features, when instead it couldlook like this:
```
"features": { 
    "FeatureName1": {
        "data":"feature_0.json",
        "type":"categorical",
        "categories":[...]
    },
    etc
}
```

Reply:
I was thinking that too... I'm not sure what the best way to handle that change would be in a way that doesn't break all of our existing datasets.

Reply:
one of 2 options:
1. fixup all existing datasets (not as hard as it sounds)
2. have the client check for the existence of `featuresMetadata` in the incoming json, for backward compatibility only

Reply:
Leigh might have some datasets lying around, I'd have to ask. I don't think anyone else has any datasets we don't know about...?

Reply:
I'll add backwards compatibility for now.

Reply:
Dataset spec updated! I'll start working on a documentation update for `colorizer-data`.